<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userView</class>
 <widget class="QWidget" name="userView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>781</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Buscar">
    <attribute name="title">
     <string>Buscar</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="emailToSearch">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 100%;
    padding: 0.75rem;
    margin: 0.75rem 0;
    border: 3px fill white;
    border-radius: 5px;
    background-color: #5e5e5e;
    color: #fff;
    font-size: 1rem;
    outline: none;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 12.5px 30px;
  border: 0;
  border-radius: 100px;
  background-color: #2ba8fb;
  color: #ffffff;
  font-weight: Bold;
font-size: 14px;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameFound">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombres: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lastnameFound">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellidos:</string>
     </property>
    </widget>
    <widget class="QLabel" name="emaiFound">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Correo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="birthdateFound">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha de nacimiento:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Publicaciones">
    <attribute name="title">
     <string>Publicaciones</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Publiaciones</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>231</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Buscar por fecha</string>
     </property>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 0;
  border-radius: 100px;
  background-color: orange;
  color: #ffffff;
  font-weight: Bold;
font-size: 14px;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Aplicar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Recorrido limitado</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Orden</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cantidad</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  border: 0;
  border-radius: 100px;
  background-color: orange;
  color: #ffffff;
  font-weight: Bold;
font-size: 14px;
border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>Aplicar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>PreOrden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>InOrden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PostOrden</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>104</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  background-color: #5e5e5e;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: 0;
  border-radius: 100px;
  background-color: orange;
  color: #ffffff;
  font-weight: Bold;
font-size: 14px;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Crear nueva publiaci√≥n</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="feed">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>471</width>
       <height>291</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>469</width>
        <height>289</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Reportes">
    <attribute name="title">
     <string>Reportes</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Reportes</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>109</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea_6">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>319</width>
        <height>109</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>691</width>
       <height>151</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>689</width>
        <height>149</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>370</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: 0;
  border-radius: 100px;
  background-color: orange;
  color: #ffffff;
  font-weight: Bold;
font-size: 14px;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Generar Reportes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fechas con m√°s publicaciones</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Publicaciones con m√°s comentarios</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BST</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Solicitudes">
    <attribute name="title">
     <string>Solicitudes</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Solicitudes</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>501</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>499</width>
        <height>109</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="users">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>501</width>
         <height>192</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Nombres</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Apellidos</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Correo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nacimiento</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Enviar solicitud</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>109</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="solicitudesRecibidas">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>311</width>
         <height>192</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Correo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Aceptar</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rechazar</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QScrollArea" name="scrollArea_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>250</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>199</width>
        <height>109</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="solicitudesEnviadas">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>211</width>
         <height>201</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Correo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuarios</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Solicitudes recibidas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>220</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Solicitudes enviadas</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Perfil">
    <attribute name="title">
     <string>Perfil</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Perfil</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: 0;
  border-radius: 100px;
  background-color: green;
  color: #ffffff;
  font-weight: Bold;
font-size: 14px;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Editar Perfil</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>230</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: 0;
  border-radius: 100px;
  background-color: red;
  color: #ffffff;
  font-weight: Bold;
font-size: 14px;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Eliminar Cuenta</string>
     </property>
    </widget>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Nombre: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lastname">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Apellidos:</string>
     </property>
    </widget>
    <widget class="QLabel" name="email">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Correo: </string>
     </property>
    </widget>
    <widget class="QLabel" name="password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Contrase√±a: </string>
     </property>
    </widget>
    <widget class="QLabel" name="birthdate">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font: bold;</string>
     </property>
     <property name="text">
      <string>Fecha nacimiento: </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: tomato;
font: bold;
font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
